contracts/colonyNetwork/ColonyNetwork.sol

```diff
@@ -102,6 +102,30 @@ contract ColonyNetwork is BasicMetaTransaction, ColonyNetworkStorage, Multicall
     emit ColonyVersionAdded(_version, _resolver);
   }
 
+  function setBridgeData(address bridgeAddress, bytes memory updateLogBefore, bytes memory updateLogAfter, uint256 gas, uint256 chainId, bytes memory skillCreationBefore, bytes memory skillCreationAfter, bytes memory setReputationRootHashBefore, bytes memory setReputationRootHashAfter) public
+  always
+  calledByMetaColony
+  {
+    if (!isMiningChain()) {
+      miningBridgeAddress = bridgeAddress;
+      require(isMiningChainId(chainId), "colony-network-can-only-set-mining-chain-bridge");
```

Comment:
I mean, they're similar, in that they both care about what the mining chain's `chainId` is, but one is concerned with the chain we're currently on, and one is concerned about the bridge data that is being set. Can you give me a code snippet to show me what you mean here, because I don't think I follow?